v2/blockstore/readwrite_test.go

```diff
@@ -405,7 +405,7 @@ func TestBlockstoreResumption(t *testing.T) {
 			require.Equal(t, wantBlockCountSoFar, gotBlockCountSoFar)
 		}
 	}
-	require.NoError(t, blockstore.CloseReadWrite(subject))
+	subject.Discard()
```

Comment:
Would be nice to have explicit test for discard now that it is a public API. 

Also separate tests for behaviour or RW blockstore after discard is called.

Reply:
done